radio_beam/beam.py

```diff
@@ -429,6 +435,16 @@ def as_kernel(self, pixscale):
                                           self.minor.to(u.deg).value/pixscale,
                                           self.pa.to(u.radian).value)
 
+    def as_struct_element(self, pixscale):
```

Comment:
AH!  I had missed that this is an `EllipticalDisk`, not an `EllipticalGaussian`.  Yes, maybe `as_boolean_kernel` or `as_elliptical_tophat_kernel`?  Or just `as_tophat_kernel`?  Is there a type of top hat that is elliptical?